setup.py

```diff
@@ -152,12 +152,13 @@ def write_version(filename: str = os.path.join(*["airflow", "git_version"])):
     'boto3>=1.7.0, <1.8.0',
 ]
 azure = [
-    'azure-storage>=0.34.0',
-    'azure-mgmt-resource>=2.2.0',
-    'azure-mgmt-datalake-store>=0.5.0',
-    'azure-datalake-store>=0.0.45',
     'azure-cosmos>=3.0.1',
+    'azure-datalake-store>=0.0.45',
     'azure-mgmt-containerinstance>=1.5.0',
+    'azure-mgmt-datalake-store>=0.5.0',
+    'azure-mgmt-resource>=2.2.0',
+    'azure-storage-blob<12.0',
```

Comment:
```suggestion
    'azure-storage-blob<12.0',  # TODO: Remove this dependency after https://github.com/snowflakedb/snowflake-connector-python/issues/225 is solved and released.
```

Reply:
@mik-laj what do you think about this?

Reply:
I agree, but I think we can add this comment when merging to avoid creating another build.

Reply:
Okay. I am fine with it if we have a ticket. I am creating a ticket.